metaseq_cli/interactive_hosted.py

```diff
@@ -187,20 +187,36 @@ def handle_exception(e):
     # pass through HTTP errors
     if isinstance(e, HTTPException):
         return e
-    # now you're handling non-HTTP exceptions only
-    response = jsonify(
-        {
-            "error": {
-                "message": str(e),
-                "type": "oops",
-                "stacktrace": traceback.format_tb(e.__traceback__),
-            }
-        }
+
+    http_code = 400 if isinstance(e, ValueError) else 500
+    return _create_error_response(
+        str(e), http_code, stacktrace=traceback.format_tb(e.__traceback__)
     )
-    if isinstance(e, ValueError):
-        response.status = 400
-    else:
-        response.status = 500
+
+
+def _validate_key(key):
+    # denylist a few placeholders various people have used
+    if key == "":
+        return False
+    if "YOUR_NAME_HERE" in key:
+        return False
+    if "$USER" in key:
+        return False
+    if "your-key-here" in key:
+        return False
+    return True
```

Comment:
Wait this is all we do for validation? lol

Reply:
before we did nothing :P